src/tangerine/vector.py

```diff
@@ -71,6 +72,11 @@ def combine_small_chunks(self, chunks):
             merged_chunks.append(buffer)
         return merged_chunks
 
+    def _load_sql_file(self, sql_file):
```

Comment:
This is a dupe of what is in search.py right? Seems like this function here is not being used anywhere?